UI/installer/mp-installer.nsi

```diff
@@ -110,16 +110,17 @@ Function PreReqCheck
 		IfSilent +1 +3
 			SetErrorLevel 3
 			Quit
-		MessageBox MB_OK|MB_ICONSTOP "This version of ${APPNAME} is not compatible with your system. Please use the 32bit (x86) installer."
+		MessageBox MB_OK|MB_ICONSTOP "${APPNAME} is not compatible with your system's architecture."
```

Comment:
Technically, it's the operating system's architecture and not necessarily the system's (hardware's) architecture. A person could, for _some_ reason, be running a 32-bit OS on 64-bit capable hardware.

Reply:
If it wasn't clear, I prefer this string:
```
"${APPNAME} is not compatible with your operating system's architecture."
```